### ISSUE TITLE
predict.py

### ISSUE THREAD

**Issue by temur3559** [temur3559/UltimateCryptoBotV2#24]
import torch
import numpy as np
import pandas as pd
from model.lstm_attention import LSTMAttentionModel
from utils.indicators import add_indicators
from data.fetch_binance import get_historical_klines

DEVICE = "cuda" if torch.cuda.is_available() else "cpu"

# Model parametrlarini sozlash
INPUT_DIM = 6  # OCHLVC + RSI, EMA, MACD (indikatorlar)
HIDDEN_DIM = 64
NUM_LAYERS = 2
OUTPUT_DIM = 1
SEQ_LENGTH = 50

# Modelni yuklash
model = LSTMAttentionModel(INPUT_DIM, HIDDEN_DIM, NUM_LAYERS, OUTPUT_DIM).to(DEVICE)
model.load_state_dict(torch.load("model/lstm_model.pth", map_location=DEVICE))
model.eval()

def preprocess_data(symbol="BTCUSDT", interval="15m"):
    df = get_historical_klines(symbol, interval, limit=SEQ_LENGTH + 50)
    df = add_indicators(df)
    df = df.dropna()
    df = df[-SEQ_LENGTH:]
    
    features = df[["close", "volume", "rsi", "ema", "macd", "macd_signal"]].values
    features = (features - np.mean(features, axis=0)) / np.std(features, axis=0)  # Normalizatsiya
    return torch.FloatTensor(features).unsqueeze(0).to(DEVICE)  # [1, SEQ, FEAT]

def predict_trend(symbol="BTCUSDT"):
    with torch.no_grad():
        x = preprocess_data(symbol)
        output = model(x)
        return "UP" if output.item() > 0.5 else "DOWN"
